app/src/App/Controller/Administrator/SendActivationEmail.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Controller\Administrator;
+
+use Demo\Domain\Service\Administrator\SendActivationEmailInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class SendActivationEmail
+{
+    public function __construct(
+        private SendActivationEmailInterface $sendActivationEmail
+    ) {
+    }
+
+    public function __invoke(Request $request): Response
+    {
+        $response = new Response();
+
+        $administratorId = $request->attributes->get('id');
+
+        $this->sendActivationEmail->execute($administratorId);
+        return $response;
```

Comment:
I would handle in the controller success & error http responses

```suggestion
        try {
              $this->sendActivationEmail->execute($administratorId);
        } cacth (\Exception $e) {
             // return http error response 
        }

        // return seccess response
```